api_app/analyzers_manager/classes.py

```diff
@@ -128,8 +131,9 @@ def _handle_analyzer_exception(self, err):
             f" Analyzer error: '{err}'"
         )
         logger.error(error_message)
-        self.report["errors"].append(str(err))
-        self.report["success"] = False
+        self.report.errors.append(str(err))
+        self.report.status = "failed"
```

Comment:
Don't hardcode strings like "failed". Instead set them like `self.report.status = Job.STATUS_CHOICES.failed`.

Reply:
Oh I see. Yeah, i'll replace that in the many places i did the exact same thing f^_^;

Reply:
Take note of: https://github.com/intelowlproject/IntelOwl/pull/498#discussion_r657763721